capstone-frontend/src/components/BookPage.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+class BookPage extends Component {
```

Comment:
nit: This can be a stateless functional component, for now at least.

Reply:
Fixed!